README.md

```diff
@@ -1,16 +1,15 @@
 # ![Base16](logo.png)
 
-An architecture for building themes based on carefully chosen syntax
+An architecture for building color schemes based on carefully chosen syntax
```

Comment:
I think this actually changes the meaning of this line. I've always interpreted this as "application themes" rather than "color schemes".

Reply:
Could it not have a double meaning here... or what about:

> An architecture for building color schemes then easily translating them to all your favorite apps - based on carefully chosen syntax

Reply:
I like it - good call.